frontend/cypress/integration/common/sidecar_injection.ts

```diff
@@ -239,32 +239,64 @@ When('I visit the overview page', function () {
 });
 
 When('I override the default automatic sidecar injection policy in the namespace to enabled', function () {
-  cy.get('[data-test=overview-type-LIST]').should('be.visible').click();
-  cy.get(`[data-test=VirtualItem_${this.targetNamespace}] button`).should('be.visible').click();
-  cy.get(`[data-test=enable-${this.targetNamespace}-namespace-sidecar-injection]`).should('be.visible').click();
-  cy.get('[data-test=confirm-traffic-policies]').should('be.visible').click();
-  ensureKialiFinishedLoading();
+  cy.request('GET', '/api/status').should(response => {
+    expect(response.status).to.equal(200);
+    const isMaistra = response.body.istioEnvironment.isMaistra;
```

Comment:
It'd be nice to get this kind of info once before any of the tests are run but I'm not sure the best way of doing that within cypress.

Reply:
me either...